### ISSUE TITLE
[Serious] Label does not convey purpose of control

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#89]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87538 |
| **Checkpoint** | 2.4 Navigable › Descriptive Labels (2.4.6.b) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 04 Work Items Workflow |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/work_items> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/05/26 at 2:21 pm |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | a3c.access@gmail.com |

## Description

The label does not convey the purpose of the control.
Pattern: the Remove Filter buttons do not convey that they remove the filter. For example, if you have set Action filters of Delete and Ingest, it announces as Action In Delete, Ingest Button, but that does not tell the user that the purpose of this button, which is to remove this filter. See Work Items User Experience Video at 12:00 and 12:25. 
Also occurs on pages 03 Objects and 06 Alerts.

## Recommendation

Rule : 
Labels MUST be meaningful.

How To Fix: 
Fix this issue by providing a descriptive label for the control.

Background: 
When form fields or interactive controls have vague or misleading labels, people may not understand the type of data expected or what a control will do if activated. This is especially true for people with cognitive disabilities and people who are blind and use a screen reader. Both the visible and the programmatic label must sufficiently describe the purpose of the form field or control. Descriptive labels give people confidence when filling out a form or using interactive content and help prevent mistakes.

## Source Code

```
<button class="badge filter-chip mb-3" data-filter-index="0">
      <strong class="mr-1">Action:</strong> IN Delete
      <span class="icon material-icons md-16" aria-hidden="true" onclick="removeFilter( 0 )">cancel</span>
    </button>
<button class="badge filter-chip mb-3" data-filter-index="1">
      <strong class="mr-1">Stage:</strong> IN Cleanup
      <span class="icon material-icons md-16" aria-hidden="true" onclick="removeFilter( 1 )">cancel</span>
    </button>
<button class="badge filter-chip mb-3" data-filter-index="2">
      <strong class="mr-1">Status:</strong> IN Failed
      <span class="icon material-icons md-16" aria-hidden="true" onclick="removeFilter( 2 )">cancel</span>
    </button>
```

## Screenshots

![04_descriptive labels](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87538/04_descriptive_labels.png)
